src/utils/constants/errorMessages.js

```diff
@@ -19,4 +19,33 @@ export const signupVerifyErrors = {
   notFound: 'No pending verification found'
 };
 
+export const loginProfileErrors = {
```

Comment:
this object name can be `userProfileErrors`. That's a bit more specific to users that `loginProfileErrors`